.github/workflows/docker-image-release.yml

```diff
@@ -55,7 +55,7 @@ jobs:
           context: ./internal/devbox/generate/tmpl/
           file: ./internal/devbox/generate/tmpl/DevboxImageDockerfile
           build-args: |
-            DEVBOX_USE_VERSION=${{ inputs.tag }}
+            DEVBOX_VERSION=${{ inputs.tag || github.ref }}
```

Comment:
A couple questions, since i'm not very familiar with this code from before. Trying to follow along the logic.
1. Why do we add the `github.ref` here? what does it signify?
2. ~Why change from `DEVBOX_USE_VERSION` to `DEVBOX_VERSION`?~ Ah, you've answered below. Cool.

Reply:
because this workflow is only triggered automatically with a tag push`github.ref` would be a tag (version).

It is a bit fragile, if we trigger it for another reason or with an invalid tag then it would create a broken image.

A better solution may be to only add `DEVBOX_USE_VERSION` for publishing non-latest (which is why I think the previous implementation with `DEVBOX_USE_VERSION=${{ inputs.tag }}` is less fragile).